web-admin/src/features/billing/plans/utils.ts

```diff
@@ -4,18 +4,18 @@ import { DateTime } from "luxon";
 import { writable } from "svelte/store";
 
 export function formatDataSizeQuota(
-  totalSizeInBytes: number,
+  sizeInBytes: number,
   storageLimitBytesPerDeployment: string,
 ): string {
   const maxSize = Number(storageLimitBytesPerDeployment);
   if (Number.isNaN(maxSize) || storageLimitBytesPerDeployment === "-1")
     return "";
-  const formattedTotal = formatMemorySize(totalSizeInBytes);
+  const formattedTotal = formatMemorySize(sizeInBytes);
```

Comment:
Nit: "total" is hard to understand in this function. What if we used the terms "usage" (the numerator) and "quota" (the denominator)? The function name could be `formatUsageOfQuota`